package/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang:1.17.6 AS helm
+FROM registry.suse.com/bci/golang:1.17 AS helm
+RUN zypper -n install git
 RUN git -C / clone --branch release-v3.8.0 --depth=1 https://github.com/rancher/helm
 RUN make -C /helm
 
-FROM alpine:3.12 AS build
+FROM registry.suse.com/suse/sle15:15.3 AS build
 ARG ARCH=amd64
-RUN apk add -U curl xz
+RUN zypper -n install curl gzip tar
```

Comment:
```suggestion
RUN zypper -n install curl gzip tar && zypper -n clean && rm -rf /var/log/*
```

Reply:
I don't think that we need to care about this one, it's just an intermediary build step.

Reply:
Right, didn't think about that.